src/Impl/CopyGitLink.Shared/Core/GitOnlineServices/GitHubnLab.cs

```diff
@@ -92,13 +92,16 @@ var relativePath
                 url += $"#L{startLineNumber + 1}";
             }
```

Comment:
If it's possible for the `endLineNumber` to be set when the `startLineNumber` isn't, it looks like that would also cause some weird effects. If not, then binding the `startLineNumber` and `endLineNumber` into a record/struct/class of some kind would allow the API to represent that neither, or both should be set.